cmd/relayer/start/start.go

```diff
@@ -58,16 +57,7 @@ func Start() *cobra.Command {
 				_, err := rly.CreateIBCChannel(override, logFileOption, seq)
 				utils.PrettifyErrorIfExists(err)
 			}
-
-			ctx, cancel := context.WithCancel(context.Background())
-			defer cancel()
-
-			updateClientsCmd := rly.GetUpdateClientsCmd()
-			utils.RunCommandEvery(ctx, updateClientsCmd.Path, updateClientsCmd.Args[1:], 3600, logFileOption)
-			relayPacketsCmd := rly.GetRelayPacketsCmd()
-			utils.RunCommandEvery(ctx, relayPacketsCmd.Path, relayPacketsCmd.Args[1:], 5, logFileOption)
-			relayAcksCmd := rly.GetRelayAcksCmd()
-			utils.RunCommandEvery(ctx, relayAcksCmd.Path, relayAcksCmd.Args[1:], 5, logFileOption)
+			go utils.RunBashCmdAsync(rly.GetStartCmd(), func() {}, func(errMessage string) string { return errMessage }, logFileOption)
```

Comment:
I think u should pass the ctx
what happens if u kill roller while `relayer start` runs?